mitodl/mit-open
# Fix frontend sentry configuration

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5109

### Description (What does it do?)
This PR fixes sentry configuration during the frontend build, with the help of some new github secrets that Mike added.

### How can this be tested?
I think we will need to do a release to actually test this. Some things worth doing:
1. Make sure the app runs locally; I changed webpack config a little
2. Run `./scripts/get_version.sh`; it should return the app version
